requirements/requirements.in

```diff
@@ -0,0 +1,6 @@
+Jinja2
```

Comment:
what's jinja for?

Reply:
It's used in a couple of dormant places in the codebase... the sql and sfdc functionality.